### ISSUE TITLE
Release for Foxy / Noetic

### ISSUE THREAD

**Issue by SteveMacenski** [OctoMap/octomap#300]
Hi, It would be great to have this released for the new ROS2 distributions. I don't see any blocking dependencies and it is blocking other packages

**Comment by wxmerkt** [comment]
I've made a release request for Foxy (https://github.com/ros/rosdistro/pull/25637). It may require a second round to get the GNUInstallDirs patches right (Dashing didn't require any; Noetic does).

Do you also need a release to Rolling?

- Octomap v1.9.3 is in Dashing: https://github.com/ros/rosdistro/blob/11b485830ef0e805edb1294906b28fcab9478212/dashing/distribution.yaml#L1612-L1630
- Octomap v1.9.5 is in Noetic: https://github.com/ros/rosdistro/blob/11b485830ef0e805edb1294906b28fcab9478212/noetic/distribution.yaml#L1942-L1960

**Comment by SteveMacenski** [comment]
Awesome thanks. If noetic needed something, chances are Foxy will too since they have mostly (or all?) the same versions of dependencies. 

Not sure we need a rolling yet, but wouldn't hurt since that's the shape of things to come.

**Comment by tylerjw** [comment]
:+1: for rolling.  I'm working on getting all the ros-planning/moveit* packages working on rolling and this package is one of our dependencies.  I should have searched and found this issue before creating a new issue.  I'll close my other issue.

**Comment by wxmerkt** [comment]
Happy to make new releases once #304 is merged to devel and a new release created. This should resolve the reported issues about registering in the ament index. If we could get some feedback that #304 works as intended and addresses the previous issue that would be great :-).

**Comment by wxmerkt** [comment]
Updated release requests for v1.9.7 which incorporate #304 are out:

- **Melodic** - v1.9.0 to v1.9.7 - https://github.com/ros/rosdistro/pull/29395
- **Noetic** - v1.9.6 to v1.9.7 - https://github.com/ros/rosdistro/pull/29396
- **Foxy** - v1.9.5 to v1.9.7 - https://github.com/ros/rosdistro/pull/29397

- **Rolling** & **Galactic** are blocked by missing keys for RHEL - see discussion in #335 

Many thanks to @ahornung for preparing the release :-).
